src/Hedgehog/Range.fs

```diff
@@ -8,9 +8,10 @@ open Hedgehog.Numeric
 type Size = int
 
 /// A range describes the bounds of a number to generate, which may or may not
-/// be dependent on a 'Size'.
+/// be dependent on a 'Size'.  The pair returned by the function contains the
+/// lower bound on the left and the upper bound on the right.
 type Range<'a> =
-    | Range of ('a * (Size -> 'a * 'a))
+    | Range of origin:'a * (Size -> 'a * 'a)
```

Comment:
```suggestion
    | Range of (origin : 'a) * (Size -> 'a * 'a)
```

Reply:
Sure.  I commited your suggestion (using GitHub's interface).

Reply:
The build failed.  I don't think that syntax is allowed.

Reply:
The [documentation for the syntax of discriminated unions](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/discriminated-unions#syntax) does not include parentheses.  I think the parentheses in the existing code are the outermost characters in a "type".

Reply:
Oh, it was my parentheses.